src/ice_spi.c

```diff
@@ -38,6 +38,7 @@ volatile static void (*g_async_callback)(volatile void *);
 volatile static void *g_async_context;
 static int dma_tx, dma_rx;
 static uint32_t dma_word;
+static bool spi_is_initialized;
```

Comment:
Somehow I forgot that we were calling this `_init()` function so many times.
It is not like we have infinite DMA channels!